### DISCUSSION TITLE
vcpkg_download_distfile and dropbox shared link

### DISCUSSION THREAD

**Discussion by yannickModulo** [microsoft/vcpkg#34429] [Q&A] [answered]
Hi,

I have big binaries proprietary third party lib. I suceeded to use Bitbucket to grab the custom registry package but the space available on bitbucket cloud is limited. I have tried to push the version on a dropbox a create a sharable link but it seems that vcpkg is unable to download the link with vcpkg_download_distfile. Is it possible ?

**Comment by autoantwort** [comment]
It should be possible, yes :)

**Comment by yannickModulo** [comment]
I have done more test and it's not working. I only succeeded to download 140Ko partial file with text inside !

**Reply by autoantwort** [reply]
Can you download it via curl? vcpkg only calls curl internally (and winhttp on windows if you don't use headers).

**Reply by yannickModulo** [reply]
It failed with Curl if no extra parameters. it works if I add  option -L (seems there is a redirection). Is there a way to add the -L to the download command ?

**Reply by autoantwort** [reply]
Not without changing the source code of vcpkg-tool

**Reply by autoantwort** [reply]
Can you not pass the redirected link to vcpkg? Or is this link not stable?

**Comment by yannickModulo** [comment, marked as answer]
I have find a way to not have a redirection using https://dl.dropboxusercontent.com instead of https://www.dropbox.com/
Thanks it's fixed!
